watson/config.py

```diff
@@ -60,6 +62,42 @@ def getboolean(self, section, option, default=False):
         val = self.get(section, option)
         return val.lower() in ('1', 'on', 'true', 'yes') if val else default
 
+    def getlist(self, section, option, default=None):
+        """
+        Return value of option in given section as a list of strings.
+
+        If option is not set, return default (defaults to an empty list).
+
+        The option value is split into list tokens using one of two strategies:
+
+        * If the value contains any newlines, i.e. it was written in the
+          configuration file using continuation lines, the value is split at
+          newlines and empty items are discarded.
+        * Otherwise, the value is split according to unix shell parsing rules.
+          Items are separated by whitespace, but items can be enclosed in
+          single or double quotes to preserve spaces in them.
+
+        Example::
+
+            [test]
+            option2 =
+                one
+                two three
+                four
+                sive six
+            option1 = one  "two three" four 'five  six'
+
+        """
+        if not self.has_option(section, option):
+            return [] if default is None else default
+
+        value = self.get(section, option)
+
+        if '\n' in value:
+            return [item for item in value.splitlines() if item]
```

Comment:
Per the [configparser docs](https://docs.python.org/2/library/configparser.html) leading whitespace is stripped from values.

I agree it's probably best to explicitly strip items in line 97 to also strip trailing whitespace.

Just say the word and I'll update the PR.